feat: fix empty incident changelog

adityathebe opened the pull request:
Resolves: https://github.com/flanksource/flanksource-ui/issues/1257
Resolves: https://github.com/flanksource/flanksource-ui/issues/1251

adityathebe commented:
![image](https://github.com/flanksource/flanksource-ui/assets/13419481/4d0a3841-ec70-49c1-b84a-a63bfa4b65d3)

moshloop commented on src/components/Changelog/IncidentHistoryItemTypeContent.tsx:
```diff
@@ -64,8 +64,26 @@ const IncidentHistoryTypeToComponentMap = new Map<
+      const evidence = incidentHistory.evidence;
+      return (
+        <>
+          added done definition{" "}
```
```suggestion
          added definition of done{" "}
```

moshloop commented:
@adityathebe still seeing empty items: https://deploy-preview-1258--flanksource-demo-stable.netlify.app/incidents/0188b8b4-c580-f527-fb29-fc5e2018417a

adityathebe commented:
> @adityathebe still seeing empty items: https://deploy-preview-1258--flanksource-demo-stable.netlify.app/incidents/0188b8b4-c580-f527-fb29-fc5e2018417a

@moshloop That's the old incident history which had the wrong type (`incident_status.updated`).
Fixed it

```
UPDATE incident_histories SET type = 'incident.status_updated' WHERE type = 'incident_status.updated'
```

moshloop commented:
- Can we change it from added evidence {name} check to just, added check {name}
- and "added DoD for check {name} 
- Remove the avatar for responders as they cause wrapping

adityathebe commented:
![image](https://github.com/flanksource/flanksource-ui/assets/13419481/8adbb1ba-889d-4528-95c6-1fc6dac3343e)

@moshloop

mainawycliffe commented:
@adityathebe can we reduce the sizes of the avatars, can be achieved by passing in the `size="sm"` attribute to the avatar

![image](https://github.com/flanksource/flanksource-ui/assets/12270550/1519047d-b31b-4750-a2b1-12d40496829b)

moshloop commented:
> @adityathebe can we reduce the sizes of the avatars, can be achieved by passing in the `size="sm"` attribute to the avatar
> 
> ![image](https://user-images.githubusercontent.com/12270550/249162657-1519047d-b31b-4750-a2b1-12d40496829b.png)

I think `sm` should be the default

adityathebe commented:
![image](https://github.com/flanksource/flanksource-ui/assets/13419481/d713be91-bc75-49bc-bd40-3bf6ae641978)

@moshloop made "sm" the default avatar size.

moshloop commented:
@adityathebe I think we want it rounded by default well
